settings.gradle

```diff
@@ -103,19 +103,28 @@ if (hasProperty("useEmbeddedTomcat"))
 
 include BuildUtils.getMinificationProjectPath(gradle)
 
+// A list of directory names or fully qualified module paths that correspond to modules to be excluded from configuration.
+// Items may be individual module names (e.g. 'luminex'), module container names (e.g. 'customModules'), or paths (e.g., ':server:modules:commonAssays:flow')
+// You must make sure to pass this list to the appropriate calls to `BuildUtils.includeModules`
+List<String> excludedModules = []
+if (hasProperty('excludedModules'))
+{
+    excludedModules.addAll("${excludedModules}".split(","))
+}
```

Comment:
Could the `BuildUtils.include*` methods automatically check the `excludedModules` property instead of having to check it in every settings file?

Reply:
I tried that, but this `hasProperty` method always returns null inside the Gradle plugin. Not sure why, but I didn't want to spend more time fighting it.